dev-tools/smoke/smoke_common.py

```
@@ -39,20 +37,17 @@ def verify_smoke_account(account_api_base, tenant_id, username, password, timeou
     for attempt in range(1, 4):
         try:
             with urllib.request.urlopen(request, timeout=timeout_seconds) as response:
```

Ruff (0.15.21)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)